### DISCUSSION TITLE
Docker Example: Is the `...` suffix in the filter flag required or a mistake?

### DISCUSSION THREAD

**Discussion by SerenModz21** [vercel/turborepo#3074] [Help] [answered]
This discussion isn't really fit the purpose of help/support as it is just a question. However, it was the only category that has uses Q&A formatting.

In the docker example, when running the build command, the filter flag has a spread operator (`...`) suffix. (eg. `--service...`). Is this intentional/required or is it a mistake? Is there a reason it is there?

https://github.com/vercel/turbo/blob/741a529ee6f51b34cfca6d881c880d7c2c0645cf/examples/with-docker/apps/api/Dockerfile#L29
https://github.com/vercel/turbo/blob/741a529ee6f51b34cfca6d881c880d7c2c0645cf/examples/with-docker/apps/web/Dockerfile#L29

**Comment by yovanoc** [comment, marked as answer]
It's for building the app and its dependencies. cf https://turbo.build/repo/docs/core-concepts/monorepos/filtering#include-dependencies-of-matched-workspaces

**Reply by SerenModz21** [reply]
So the spread operator (`...`) suffix is required. Thank you so much!
